docs/contributors/code/release.md

```diff
@@ -71,19 +74,21 @@ Guidelines for proof-reading include:
 -   Remove mobile app pull request entries.
 
 #### 2. Selecting the release highlights
-Once you have cleaned up the changelog, the next step is to choose a few changes to highlight in the release post. These highlights usually focus on new features and enhancements, including performance and accessibility ones, but can also include important API changes or critical bug fixes.
+Once the changelog is cleaned up, the next step is to choose a few changes to highlight in the release post. These highlights usually focus on new features and enhancements, including performance and accessibility ones, but can also include important API changes or critical bug fixes.
 
-Given the big scope of Gutenberg and the high number of pull requests merged in each milestone, it is not uncommon to overlook impactful changes worth highlighting; it is recommended to make this step a collaborative effort and share your picks with the Gutenberg Core team.
+Given the big scope of Gutenberg and the high number of pull requests merged in each milestone, it is not uncommon to overlook impactful changes worth highlighting; because of this, this step is a collaborative effort between the release manager and Gutenberg Core team members.
 
 #### 3. Drafting the release post
 
+Because of the nature of the release post content, responsibilities are divided in this step. While the post **can either be written by the release manager or delegated to another core member** agreed upon in advance, **visual assets should be created by the design team**.
```

Comment:
The goal of stressing this is to clarify that these are **not** requirements to perform a release: the post can be written by somebody else if the release manager doesn't feel comfortable writing, whereas design folks can provide awesome assets. Rewrites to convey this message are welcome.

Reply:
I wouldn't say "should". I'd say "visual assets are created by the design team." If possible, would be good to make it  bit clearer where to get help (could be as simple as going to the #design channel).

Reply:
Yeah, this comment got outdated as the edit didn't feel right and later switched it to:

> visual assets are best created by the design team

But your suggestion is even better 💯